src/moh_data/main.py

```diff
@@ -77,3 +184,18 @@ def export_data(data, filename):
         run_data.plot_cumulative_sum()
         run_data.plot_daily_arrival_sum()
         run_data.plot_overseas_date_reported()
+
+        print('Confirmed cases on 2020-04-14 = ', run_data.get_confirmed_cases_on((4, 14)))
```

Comment:
I get the following when executing the above command:
```
>>> print('Confirmed cases on 2020-04-14 = ', run_data.get_confirmed_cases_on((4, 14)))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/Alan/virtualenv/moh-data/lib/python3.7/site-packages/moh_data/main.py", line 44, in get_confirmed_cases_on
    return self._get_cases_on(self._total_daily_confirmed, date)
  File "/Users/Alan/virtualenv/moh-data/lib/python3.7/site-packages/moh_data/main.py", line 121, in _get_cases_on
    return int(sheet[sheet.index == datetime(2020, date[0], date[1])].get_values().flatten()[0])
  File "/Users/Alan/virtualenv/moh-data/lib/python3.7/site-packages/pandas/core/generic.py", line 5274, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'get_values'
```

Reply:
Could you please check which version of `pandas` you have installed?

Reply:
Here are the packages I am using for moh-data:
```
Package         Version
--------------- ----------
beautifulsoup4  4.9.0
certifi         2020.4.5.1
chardet         3.0.4
cycler          0.10.0
idna            2.9
kiwisolver      1.2.0
matplotlib      3.2.1
moh-data        0.1.0
numpy           1.18.2
pandas          1.0.3
pip             20.0.2
pyparsing       2.4.7
python-dateutil 2.8.1
pytz            2019.3
requests        2.23.0
setuptools      46.1.3
six             1.14.0
soupsieve       2.0
urllib3         1.25.8
wheel           0.34.2
xlrd            1.2.0
```

Reply:
Aha. Seems like `get_values()` has been deprecated in newer versions of pandas. We can just use `values` instead. Will fix this.

Reply:
Cool, thanks.